arsdk-rs/src/command.rs

```diff
@@ -1,11 +1,12 @@
 use crate::common;
 use crate::frame::Data;
 use crate::jumping_sumo;
+use crate::ardrone3::ArDrone3;
 #[derive(Debug, PartialEq, Clone, Copy)]
 
 pub enum Feature {
```

Comment:
IDK if we should handle the enum like that, but we can definitely figure it out later.
This works for now, although the `ArDrone3` doesn't have all options/class and etc